docs_app/content/tutorial/basics.md

```diff
@@ -0,0 +1,176 @@
+# The basics
+
+## Converting to observables
+
+<!-- skip-example -->
+
+```ts
+import { Observable } from 'rxjs';
+
+// From one or multiple values
+const ofObser = Observable.of('foo', 'bar');
```

Comment:
could you use the static creation operators and the pipable operators in the examples